ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__far-edge-vran-ptp-4.14.yaml

```diff
@@ -0,0 +1,60 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: ci
+    namespace: openshift
+    tag: golang-1.20
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: eco-ci-cd
+  cluster: build05
+  cron: 00 12 * * 1
+  steps:
+    env:
+      bypass_olm_sync: "false"
+      cnf_polarion_branch: enabling_updating_exsisting_test_run
+      cnf_polarion_repo: https://gitlab.cee.redhat.com/alugasi/cnf-polarion.git
+      deploy_du: "true"
+      du_pull_spec: quay.io/openshift-release-dev/ocp-release:4.15.14-x86_64
+      launch_description: telco ran tests
+      launch_name: telco-ran
+      lock_label_host: registry.kni-qe-41.lab.eng.tlv2.redhat.com
+      mirror_spoke_operator_images: "true"
+      ocp_edge_branch: master
+      ocp_edge_repo: https://gitlab.cee.redhat.com/ocp-edge-qe/ocp-edge.git
+      olm_ansible_extra_args: ocp_minor_version%3A%20%2715%27
```

Comment:
Maybe we should add a comment about the format used here for the values?

Reply:
I tried that, 
Comments are deleted when you run make update